packages/generator-volto/README.md

```diff
@@ -63,7 +65,16 @@ Arguments:
 
 to see a full list of options and arguments.
 
-You can provide an specific Volto version like:
+> [!NOTE]  
+> Besides the Volto project the generator also generates a theme add-on inside `src/addons`
+> with the default name of `volto-[<project-name>]`.
+> You can provide a specific name for the theme add-on that is created on project generation like:
+> ```bash
+> yo @plone/volto --defaultAddonName="volto-project-theme"
+> ```
+> If you've used the example project name `myvoltoproject` inside `src/addons` you will find the Volto theme add-on named `volto-project-theme` instead of the default generator name `volto-myvoltoproject`.
```

Comment:
```suggestion
> If you've used the example project name `myvoltoproject` inside `src/addons`, you will find the Volto theme add-on named `volto-project-theme` instead of the default generator name `volto-myvoltoproject`.
```